### DISCUSSION TITLE
Flying multiple crazyflie experiencing crashes or not taking off problems

### DISCUSSION THREAD

**Discussion by Sherry-97** [USC-ACTLab/crazyswarm#532] [Q&A]
I first try to position the flight of a single crazyflie under optitrack. I strictly follow the following rules: 
1. Create the rigid body under optitrack with the front of the crazyflie pointing to the x-axis and propagate it in a marked based manner; 2. Assign addresses to the crazyflie and compare it with optitrack and yaml file. 
So we get a good flying effect when a single crazyflie is flying. However, problems were encountered when flying multiple crazyflies.
First my yaml config file, 1. **I have a crazyradio pa and have assigned the same channel to both crazyflie, is this the correct setting?** 
![Selection_008](https://user-images.githubusercontent.com/70084772/141947528-2267edb2-9588-4e67-be94-c9e7d4bdd5bf.png)
2. I am getting an error when starting hover_swarm, **Couldn't open joystick**, which I also seem to get when flying a single crazyflie, as if it should not be flown? Because I can see the location of crazyflie in rviz; 
![Selection_007](https://user-images.githubusercontent.com/70084772/141947886-7e730ab3-ba1d-456e-9507-fc7d4d4635cb.png)
![Selection_004](https://user-images.githubusercontent.com/70084772/141947674-1175ca42-e9b8-4eaf-91fa-5fd88e923b49.png)
86f0ee.png)
3. The next thing is that I encounter py files and crazyflies either crash or fail to take off
![Selection_006](https://user-images.githubusercontent.com/70084772/141947788-4746b4c8-2928-4db6-a813-2b83e6d60e6d.png)
![Selection_003](https://user-images.githubusercontent.com/70084772/141947793-010448ff-c618-43f8-9cfd-e7f98b07d39a.png)

This looks like a communication problem? I would like to get advice on how to go about troubleshooting these. Thanks a lot!

**Comment by whoenig** [comment]
1. Yes this is correct.
2. Not to worry, unless you actually plugged in a joystick.
3. Also everything normal in the logs.

Since you see crashes, make sure that the communication works. If you have the LED ring it should turn green. Otherwise enable logging and logging_pose in the launch file and make sure that /cf1/pose matches your expectation.
